README.md

```diff
@@ -51,7 +51,7 @@ _Modular_ means that all the building blocks can be easily replaceable.
    * [ ] Docker on Linux _(optional)_
    * [ ] SGX on Graphene _(optional)_
 1. Payment platform
-    * [x] **Payments with NGNT**
+    * [x] **Payments with Erc-20**
```

Comment:
This kinda looks the same as 2 lines below. Maybe it should just be removed? :thinking:

Reply:
`erc-20` is indeed duplicate, but i think its wrong: it refers to "driver" or "feature", while it should be "our token".

The line should say `Payments with GLM`, then its not a duplicate